internal/k8s/status.go

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package k8s contains helpers for setting the IngressRoute status
```

Comment:
I think this package might be better named `ingressroute` given it no longer contains watchers. I can do this in a follow-up PR